src/main/resources/games/strategy/engine/xml/game.dtd

```diff
@@ -81,6 +81,8 @@
 		name ID #REQUIRED
 		optional (true | false) 'false'
 		canBeDisabled (true | false) 'false'
+		isDefaultAI (true | false) 'false'
+		isNonPlayable (true | false) 'false'
```

Comment:
Looking at `isNonPlayable` in conditionals, we use it three times in this branch, twice negated and once not, eg: https://github.com/triplea-game/triplea/pull/1824/files#diff-b161056da98b1007729f14654e98597bR466

`!isNonPlayable` seems like a double negation. Thinking about alternatives to avoid the double negation, I see that we need the default value to be 'false'. How about 'hiddenPlayer' instead of 'isNonPlayable'.

`!isHiddenPlayer` IMO is a bit easier to grok by avoiding the double negation problem of `!isNonPlayable`.